### DISCUSSION TITLE
Budibase developers for hire

### DISCUSSION THREAD

**Discussion by gillima99** [Budibase/budibase#3466] [General]
Hi, 
I want to get a small crud application done using Budibase and I have looked for freelance developers on Upwork and Freelancer but cannot find any. 
Not sure if this is the place to look if anyone is interested?
Thanks,
Mark.

P.S. happy to share the app when its done.

**Comment by shreyfirst** [comment]
I'd be happy to work on a project. Email me the details: shreysgupta [at] gmail.com

**Comment by ryzam** [comment]
Hi @gillima99 , I also interested to help you as freelancer to build your CRUD app. Can email to me irwanazam[at]gmail.com

**Comment by gillima99** [comment]
Hi both, 
I sent you both emails but received no replies?
Both still not interested?
Mark.

**Reply by ryzam** [reply]
sorry, already check but not received any email. Can you re-send again

**Reply by gillima99** [reply]
re-sent to irwanazam[at]gmail.com

**Comment by ryzam** [comment]
Hi Mark

Actually frankly speaking I'm new to BudiBase and not much experience. Most
of my experience comes from my current role as software developer. So I
don't have any reference apps or any public app yet.

Thanks
Irwan

**Comment by VikingDadMedic** [comment]
@gillima99 Hey Mark! I am actually just finishing up a crud application (with automations) across multiple databases - using BudiBase as the "glue". So if you're still in need of a dev with BB experience, feel free shoot me an email (ryan` @ `arqiteqt.io) so we can discuss your apps requirements.

**Comment by itworksmart** [comment]
@gillima99 would like to know what kind of BB app you need.
If still need someone please mail me via my id at gmail.
Thanks

**Comment by JayP718** [comment]
Hi,

Feel free to reach out. I've built multiple plugins, won the Budibase hackathon and built multiple Budibase applications.

Jay@a3datasolutions.com
